lib/openstudio/extension/runner_config.rb

```diff
@@ -40,17 +43,31 @@ def add_config_option(name, value)
         @data[name.to_sym] = value
       end
 
+      def self.get_local_bundle_config_path(dirname)
+        # Bundler.settings reads from pwd, and I can't test with this...
+        bundler_settings = Bundler::Settings.new(File.join(dirname, '.bundle'))
+
+        if (bundle_install_path = bundler_settings.path.explicit_path)
+          puts "Defaulting bundle_install_path to bundle's local config value: '#{bundle_install_path}'"
+        else
+          puts "Defaulting to .bundle/install (and ignoring system wide: Bundler.configured_bundle_path.base_path=#{bundler_settings.path.base_path})"
+          bundle_install_path = File.join(dirname, '.bundle/install/')
+        end
+
+        return bundle_install_path
+      end
+
       ##
       # Return the default runner configuration
-      def self.default_config
+      def self.default_config(dirname)
         return {
           file_version: '0.1.0',
           max_datapoints: 1E9.to_i,
-          num_parallel: 2,
+          num_parallel: Parallel.processor_count - 1,
```

Comment:
```suggestion
          num_parallel: Parallel.processor_count,
```

If we want to revert back to using all threads